shared/uavobjectdefinition/hwsprf3e.xml

```diff
@@ -0,0 +1,142 @@
+<?xml version="1.0"?>
+<xml>
+	<object name="HwSprf3e" singleinstance="true" settings="true" category="HardwareSettings">
+		<description>Selection of optional hardware configurations.</description>
+		<field name="Uart1" units="function" type="enum" elements="1" parent="HwShared.PortTypes" defaultvalue="Disabled">
+			<description/>
+			<options>
+				<option>Disabled</option>
+				<option>DSM</option>
+				<option>SRXL</option>
+				<option>HoTT SUMD</option>
+				<option>HoTT SUMH</option>
+				<option>S.Bus</option>
+				<option>S.Bus Non Inverted</option>
+				<option>IBus</option>
+				<option>ComBridge</option>
+				<option>DebugConsole</option>
+				<option>FrSKY Sensor Hub</option>
+				<option>FrSKY SPort Telemetry</option>
+				<option>GPS</option>
+				<option>HoTT Telemetry</option>
+				<option>I2C</option>
+				<option>LighttelemetryTx</option>
+				<option>MavLinkTX</option>
+				<option>MavLinkTX_GPS_RX</option>
+				<option>MSP</option>
+				<option>OpenLog</option>
+				<option>Storm32Bgc</option>
+				<option>Telemetry</option>
+				<option>TBS Crossfire</option>
+			</options>
+		</field>
+		<field name="Uart2" units="function" type="enum" elements="1" parent="HwShared.PortTypes" defaultvalue="S.Bus">
```

Comment:
We don't currently have a way to conditionally override defaults but it would sure be nice for a lot more than just this. I wonder if there's a way to tell the difference between SPRF3 and the Frsky board (any pinout variation?)?